### DISCUSSION TITLE
How to use this driver for MFA or SSO?

### DISCUSSION THREAD

**Discussion by Jie89** [aws/amazon-redshift-python-driver#191] [Q&A] [answered]
I can use the driver with a service account successfully. Now the question is how would I use the driver with a SSO account that needs an MFA? Anyone have some sample code to share? Thanks!

**Comment by Brooke-white** [comment, marked as answer]
Hey @Jie89 , we have some examples of using redshift_connector [in the tutorials]( https://github.com/aws/amazon-redshift-python-driver/blob/master/tutorials/001%20-%20Connecting%20to%20Amazon%20Redshift.ipynb). Also you may check the [AWS docs](https://docs.aws.amazon.com/redshift/latest/mgmt/python-connect-examples.html) for examples. If the existing documentation doesn't cover your use case please let us know more about your specific use case here or using the AWS docs feedback button [on this page](https://docs.aws.amazon.com/redshift/latest/mgmt/python-connect-examples.html). If you have additional questions please let us know. AWS Support can provide additional guidance as needed.
